sanic/server/socket.py

```diff
@@ -112,6 +112,17 @@ def configure_socket(
                 server_settings["port"],
                 backlog=backlog,
             )
+        except PermissionError:
+            p = server_settings["port"]
+            if not p or p >= 1024:
+                raise
+            addr = f"{server_settings['host']}:{p}"
```

Comment:
Do we care about ipv6 brackets here?

Reply:
I should probably add a function in sanic/helpers for that and port numbers as these are getting repeated in many places.

Reply:
Are you planning to do that here, or should we make it a new issue to handle on its own?